http.go

```diff
@@ -17,11 +17,12 @@ type resultContainer struct {
 	Success  bool      `json:"success"`
 	Message  string    `json:"message"`
 	Comments []Comment `json:"comments,omitempty"`
+	Count    *int       `json:"count,omitempty"`
 }
 
 func (res *resultContainer) render(w http.ResponseWriter) {
 	if res == nil {
-		res = &resultContainer{false, "Some internal error occurred", nil}
+		res = &resultContainer{false, "Some internal error occurred", nil, nil}
```

Comment:
Yep, we can have this as a value. Or better yet, initialize this as 0.

If you're worried about `omitempty` removing it because it's 0, we can handle that in front-end.